pylint_django/checkers/foreign_key_strings.py

```diff
@@ -95,7 +92,12 @@ def open(self):
             # this means that Django wasn't able to configure itself using some defaults
             # provided (likely in a DJANGO_SETTINGS_MODULE environment variable)
             # so see if the user has specified a pylint option
-            if self.config.django_settings_module is None:
+            if hasattr(self, "linter"):
+                django_settings_module = self.linter.config.django_settings_module
```

Comment:
This line is hit at my end (`self.config` is `None`) meaning that at line 113 the `self.config.django_settings_module` is resulting in an error, I think that 113 should be: `settings.configure(Settings(django_settings_module))` which worked for me.